src/drivers/dw/ssi-spi.c

```diff
@@ -120,6 +120,7 @@ struct spi_dma_config {
 	enum spi_xfer_direction dir;
 	void *src_buf;
 	void *dest_buf;
+	uint8_t buffer_size;
```

Comment:
Why uint8_t? In the spi struct you have defined it as uint32_t.

Reply:
changed